packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -55,7 +54,7 @@ class AtLookupImpl implements AtLookUp {
   AtChops? _atChops;
 
   AtLookupImpl(String atSign, String rootDomain, int rootPort,
-      {this.privateKey,
```

Comment:
keep this in constructor for backward compatibility

Reply:
updated